.github/workflows/docs.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           PIP_EXTRA_INDEX_URL: https://download.pytorch.org/whl/cpu
         run: |
           conda config --set notify_outdated_conda false
-          nox -e ${{ matrix.example-name }}
+          nox -e ${{ matrix.example-name }} -- --no-website
```

Comment:
Why is this?

Reply:
When running `nox -e some-example`, we want to both generate the rst files from the python scripts, and to turn this rst into HTML for local visualization. On CI the second step is a separate CI job, so we can skip the rst => HTML transformation in this step and save some time.